api/src/test/java/io/opencensus/trace/TracerTest.java

```diff
@@ -94,6 +94,31 @@ public void run() {
     assertThat(noopTracer.getCurrentSpan()).isSameAs(BlankSpan.INSTANCE);
```

Comment:
Should this test also check that `end` wasn't called in the wrapped Runnable?

Reply:
Done.